### PR TITLE
Font Style Updates, Part Three: Threads

### PR DESCRIPTION
**Pull request by joshuatbrown**
#939 

Updated the Threads view with font styles to match Figma.

| Top | Bottom |
|--------|--------|
| Simulator Screenshot - iPhone 15 Pro - 2024-03-15 at 11 43 42 | Simulator Screenshot - iPhone 15 Pro - 2024-03-15 at 11 43 49 |

### CODE REVIEW

**Review by Chardot — COMMENTED** [review]
@joshuatbrown Are the colors of the background gradient in note cards updated here? I don't see it under file changes, but maybe you changed it as part of a different PR.

**Comment by joshuatbrown** [comment]
@Chardot I haven't updated colors yet -- that will be done in #938.

**Comment by Chardot** [comment]
> @Chardot I haven't updated colors yet -- that will be done in #938.

Ok! Approving this as it looks great 👍

**Review by Chardot — APPROVED** [review]
Looks great!

File: Nos/Views/AuthorLabel.swift [hunk 1]
```diff
@@ -22,9 +22,9 @@ struct AuthorLabel: View {
     var body: some View {
         HStack {
             AvatarView(imageUrl: author.profilePhotoURL, size: 24)
-            Text(attributedAuthor)
+            SwiftUI.Text(attributedAuthor)
```
Comment (joshuatbrown) [inline Nos/Views/AuthorLabel.swift:25]: Wasn't able to use `PlainText` here (or below) for some reason, but it'll be gone soon anyway, so I'm not too concerned.

Comment (martindsq) [reply Nos/Views/AuthorLabel.swift:25]: It might be because of the constructor. I bet that PlainText didn't define a constructor for AttributedString.

Comment (joshuatbrown) [reply Nos/Views/AuthorLabel.swift:25]: You're probably right.

**Review by martindsq — APPROVED** [review]

**Pull request merged**